gallery/src/styles/CssVariables/CssVariablesPage.tsx

```diff
@@ -178,6 +178,19 @@ const CssVariablesPage = () => {
               , making its z-index irrelevant.
             </NxWarningAlert>
           </PropertyDocItem>
+          <PropertyDocItem propertyVar="--nx-z-index-drawer">
+            <NxP>
+              The z-index of <NxCode>NxDrawer</NxCode> component. This is provided as a variable in
+              case downstream code wants to set the z-index of another element relative to it (e.g. to ensure that
+              that other element always appears above, or always below, the drawer).
+            </NxP>
+            <NxWarningAlert>
```

Comment:
Pretty sure this isn't true for `NxDrawer` since it isn't modal.

Reply:
> Pretty sure this isn't true for `NxDrawer` since it isn't modal.

Fixed here: 8a764eccfcc40518141ac3c13fd290c7b2c33060